tests/base.py

```diff
@@ -0,0 +1,11 @@
+import maya.cmds as mc
+from unittest import TestCase
+
+
+class BaseTestCase(TestCase):
```

Comment:
Good idea to create this class in the first place. Might come in handy for something other than the setUp/tearDown stuff as well. Thanks!!

Reply:
Yeah I find that having a base TestCase class is almost always handy for projects!  Marking this as resolved.